app/Http/Controllers/User/BlogController.php

```diff
@@ -14,6 +14,36 @@ class BlogController extends Controller
      */
     public function index()
     {
-        return view('user.index');
+        $blogs = Blog::all();
+        if ($this->defineUser()->can_delete === Blog::IS_TRUE) {
+            $canDelete = true;
+        }else {
+            $canDelete = false;
+        }
+        $canSee = $this->defineUser()->can_see;
+        if($canSee === Blog::IS_TRUE)
+            return view('user.blog.index', ['blogs'=>$blogs, 'canDelete'=>$canDelete]);
+        else
+            return redirect()->back();
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $request->validate([
+            'title' => 'required',
+            'content' => 'required',
+        ]);
+        $blog = new Blog;
+        $blog->user_id = $this->defineUser()->id;
+        $blog->title = $request->title;
+        $blog->content = $request->content;
+        $blog->save();
```

Comment:
Not sure if delete() is successful
=> So need to check more error cases (use IF || Try .. cath)

Reply:
@nguyentienthinh can you resolve in new pull request.

Reply:
dạ cái này em đang fix ở branch của em á Anh

Reply:
wait for later updates